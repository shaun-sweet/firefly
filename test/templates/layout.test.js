import Vue from 'vue'
import { shallow } from 'vue-test-utils'

import Layout from '../../templates/layout.vue'


describe('Layout template', () => {
  it('should render the default layout template', () => {
    const wrapper = shallow(Layout)
    expect(wrapper.element).toMatchSnapshot();
  })
})
