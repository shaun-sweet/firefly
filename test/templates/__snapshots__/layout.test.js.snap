// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout template should render the default layout template 1`] = `
<q-layout>
  <q-toolbar>
    <q-toolbar-title>
      
      Title
    
    </q-toolbar-title>
  </q-toolbar>
   
  <router-view />
</q-layout>
`;
